File: AccuPay/Domain/JobLevels/JobPoint.vb
Option Strict On

Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Imports Acupay
Imports AccuPay.Entity

Namespace Global.AccuPay.JobLevels

    <Table("jobpoint")>
    Public Class JobPoint

        Public Property RowID As Integer?

        Public Property OrganizationID As Integer?

        Public Property EmployeeID As Integer?

        Public Property OccurredOn As Date

        Public Property Points As Integer

        Public Property Comments As String

        <ForeignKey("EmployeeID")>
        Public Overridable Property Employee As Employee

    End Class

End Namespace

File: AccuPay.Core/Entities/JobPoint.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Core.Entities
{
    [Table("jobpoint")]
    public class JobPoint
    {
        public int? RowID { get; set; }

        public int? OrganizationID { get; set; }

        public int? EmployeeID { get; set; }

        public DateTime OccurredOn { get; set; }

        public int Points { get; set; }

        public string Comments { get; set; }

        [ForeignKey("EmployeeID")]
        public virtual Employee Employee { get; set; }
    }
}